lessons/basics/composition.md

```diff
@@ -21,11 +21,11 @@ Modules are the best way to organize functions into a namespace.  In addition to
 Let's look at a basic example:
 
 ``` elixir
-defmodule Example do
-  def greeting(name) do
-    ~s(Hello #{name}.)
-  end
-end
+iex> defmodule Example do
```

Comment:
@doomspork remove d302e14 and recommit to my same branch or resubmit another pull request in different branch? Please advise.